index.html

```diff
@@ -0,0 +1,153 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>concert page</title>
+    <link rel="stylesheet" href="css/index.css">
+</head>
+<body>
+    <main>
+      <!-- start of header -->
+        <header>
+          <a href="index.html" class="logo-header desktop-v">
+            <div id="logo">
+              <img src="img/musical-festival-02.jpg" alt="image of a concert">
+              <h2>concerts 2022</h2>
+            </div>
+          </a>
+          <nav class="navbar desktop-v">
+            <ul>
+              <a href="about-us.html">
+                <li>About</li>
+              </a>
+              <a href="#">
+                <li>Program</li>
+              </a>
+              <a href="#">
+                <li>Join</li>
+              </a>
+              <a href="#">
+                <li>Sponsor</li>
+              </a>
+              <a href="#">
+                <li>News</li>
+              </a>
+            </ul>
+            <div class="campain desktop-v">campain</div>
+          </nav>
+          <nav class="navbar mobile-v">
+              <ul>
+                <a href="index.html"><li>home</li> </a>
+                <a href="about-us.html"><li>about us</li> </a>
+              </ul>
+          </nav>
+        </header>
+      <!-- end of header -->
+      <!-- start of home section -->
+        <section class="home-section">
+          <div class="home-wrapper">
+
+            <div class="burger-btn mobile-v">
+              <span class="bar"></span>
+              <span class="bar"></span>
+              <span class="bar"></span>
+          </div>
+
+            <h1 class="home-h1">
+                <span>hello world</span><br>
+                welcome to concert #20 of 2022
+            </h1>
+            <p class="home-p">
+                join us on the celebretion of this 2022 on summer.
+                it will take place on toronto canada.<br>
+                !Hope seeing you there!
+            </p>
+            <h2 class="home-h2">
+                2022.8.22 ~ 2022.8.23 <br>
+               <span> @CEO alejandro torres - venezuela </span>
+            </h2>
+          </div>
+        </section>
```

Comment:
on the h1 it's not a color it's an image that's why I change it. so that's why I will not change it, even though I will change the width of it to match it, is that okay?